core/src/main/java/org/apache/calcite/sql/SqlBasicTypeNameSpec.java

```diff
@@ -179,6 +180,10 @@ public String getCharSetName() {
       writer.keyword("WITH LOCAL TIME ZONE");
     }
 
+    if (isWithTimeZone) {
```

Comment:
nit: Can this be made an "else if" instead?

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/216/commits/5d7d7ba9b0ffbbb91244031110a62213420709aa